lib/js/dom.nim

```diff
@@ -1297,3 +1307,11 @@ proc offsetHeight*(e: Node): int {.importcpp: "#.offsetHeight", nodecl.}
 proc offsetWidth*(e: Node): int {.importcpp: "#.offsetWidth", nodecl.}
 proc offsetTop*(e: Node): int {.importcpp: "#.offsetTop", nodecl.}
 proc offsetLeft*(e: Node): int {.importcpp: "#.offsetLeft", nodecl.}
+
+
+since (1, 3):
+  func newDOMParser*(): DOMParser {.importcpp: "(new DOMParser()​​)".}
+    ## DOM Parser constructor.
+
+  func parseFromString*(this: DOMParser; str: cstring; mimeType: cstring): Document {.importcpp.}
```

Comment:
still meh on that; at least provide an enum to make life easier for users eg:
type MimeTypeDom = enum
  textHtml = "text/html" etc

Reply:
@timotheecour but don't we already have that? https://nim-lang.org/docs/mimetypes.html

Reply:
Please don't. JS grows relatively fast and change really quickly, so we should not spent too much time chasing these things. Different browsers might support more MIME types, and it's not really our place to dictate how to use an API.

Reply:
I agree with alaviss on this point.

Reply:
fine, we can later introduce
```nim
type MimeTypes = enum
  textHtml = "text/html".cstring
  # etc
```
and use it as:
`let a = prsr.parseFromString("<html>...</html>".cstring, $textHtml)`

(but not sure what's the difference with `fetch` API case https://github.com/nim-lang/Nim/pull/12531/files#r403861811
> So what, please use enums.
)